Packs/Venafi/Integrations/VenafiV2/VenafiV2.py

```diff
@@ -0,0 +1,371 @@
+        set_integration_context({
+            'token': token,
+            'refresh_token': refresh_token,
+            'expire_date': str(expire_date)
+        })
+
+    def _get_certificates(self, args: dict[str, Any]) -> dict:
+        """
+        This method creates the HTTP request to retrieve the certificates the user has.
+
+        Args:
+            args (dict): The arguments for the command passed to the request.
+
+        Returns:
+            dict: The certificates.
+        """
+
+        headers = {
+            'Authorization': f'Bearer {self.token}'
+        }
+
+        certificates = self._http_request(
+            method='GET',
+            url_suffix='/vedsdk/certificates/',
+            headers=headers,
+            params=args
+        )
+
+        return certificates
+
+    def _get_certificate_details(self, guid: str) -> dict:
+        """
+        This method creates the HTTP request to retrieve certificate details.
+
+        Args:
+            guid (str): The GUID of the certificate.
+
+        Returns:
+            dict: The certificate details.
+        """
+
+        headers = {
+            'Authorization': f'Bearer {self.token}'
+        }
+
+        url_suffix = f'/vedsdk/certificates/{guid}'
+        certificate_details = self._http_request(
+            method='GET',
+            url_suffix=url_suffix,
+            headers=headers
+        )
+
+        return certificate_details
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client: client to use
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    message: str = ''
+    try:
+        test_empty_args: Dict = {}
+        results = client._get_certificates(test_empty_args)
+        if results:
+            message = 'ok'
+    except DemistoException as e:
+        if 'Forbidden' in str(e) or 'Authorization' in str(e):
+            message = 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
+    return message
+
+
+def get_certificates_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """
+    Get all the certificates belong to a user.
+
+    Args:
+        client (Client): A Venafi client.
+        args (dict): The arguments for the command passed to the request.
+    Returns:
+        A CommandResult object with an outputs, raw_response and readable table, in case of a successful action.
+    """
+
+    outputs: dict[str, Any] = {}
+    response = client._get_certificates(args)
+    if response:
+        outputs = delete_links_from_response(response)
```

Comment:
If no response, does it mean no certificates at all were returned?

Also, instead of this if, we can just pass to the outputs entry of command results the `delete_links_from_response(response)` - just make sure this method can deal with None (returns empty dict or something)